test/controllers/api/v1/tasks_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+
+class Api::V1::TasksControllerTest < ActionController::TestCase
+  test 'should get show' do
+    author = create(:user)
+    task = create(:task, author: author)
+    get :show, params: { id: task.id, format: :json }
+    assert_response :success
+  end
+
+  test 'should get index' do
+    get :index, params: { format: :json }
+    assert_response :success
+  end
+
+  test 'should post create' do
+    author = create(:user)
+    sign_in(author)
+    assignee = create(:user)
+    task_attributes = attributes_for(:task).
+      merge({ assignee_id: assignee.id })
+    post :create, params: { task: task_attributes, format: :json }
+    assert_response :created
+
+    data = JSON.parse(response.body)
+    created_task = Task.find(data.dig('task', 'id'))
+
+    assert created_task.present?
+    assert_equal task_attributes.stringify_keys, created_task.slice(*task_attributes.keys)
+  end
+
+  test 'should put update' do
+    author = create(:user)
+    assignee = create(:user)
+    task = create(:task, author: author)
+    task_attributes = attributes_for(:task).
+      merge({ author_id: author.id, assignee_id: assignee.id }).
+      stringify_keys
+      
+    patch :update, params: { id: task.id, format: :json, task: task_attributes }
+    assert_response :success
+
+    task.reload
+    assert_equal task.slice(*task_attributes.keys), task_attributes
+  end
+
+  test 'should delete destroy' do
+    author = create(:user)
+    task = create(:task, author: author)
+    delete :destroy, params: { id: task.id, format: :json }
+    assert_response :success
+
+    assert !Task.where(id: task.id).exists?
```

Comment:
Ruby has methods that allow us to avoid bangs and improve code readability.
`assert Task.find(id).blank?`